app/src/main/java/com/github/siela1915/bootcamp/CommentAdapter.java

```diff
@@ -20,6 +22,7 @@ public class CommentAdapter extends RecyclerView.Adapter<CommentViewHolder> {
     List<Comment> comments;
```

Comment:
It would be better if the attributs are private and create getters and setters if needed